index.html

```diff
@@ -2,10 +2,43 @@
 <html>
   <head>
     <title>Relaxr</title>
+    <link href="https://fonts.googleapis.com/css?family=Open+Sans:300,600,800" rel="stylesheet" type="text/css">
     <link rel="stylesheet" type="text/css" href="css/style.css">
   </head>
   <body>
     <header>
+    	<h1>Relaxr</h1>
+    	<center>
+    	<h2>Get piece of mind with a single tap</h2>
+    	<a href="#" class="button1">Get it Now</a></center>
     </header>
+   	<article class="Benefits>">
+    	<h3>Benefits</h3>
+	    	<p class="b1">The perfect personal assistant. Relaxr does your job for you so you can enjoy life as it is meant to be.</p>
+		<ul>
+			<li>Schedule meetings for you</li>
+			<li>Excel automation to complete your work for you</li>
+			<li>Responds to emails on your behalf</li>
+			<li>Does all your work for you with our revolutionary AI technology</li>
+		</ul>
+	</article>
```

Comment:
Knowing what HTML5 tag to use is really tough. It almost philosophical, you have to ask yourself. OK, I am an article or a section. It's like, "Who am I." In the instance, I think you would be better with a section though. The content doesn't feel much like an article.